WooCommerce/Classes/ViewRelated/Blaze/BlazeEditAd/BlazeEditAdView.swift

```diff
@@ -252,6 +255,16 @@ private extension BlazeEditAdView {
                 value: "Change image",
                 comment: "Change image button title in the Blaze Edit Ad screen."
             )
+            static let imageSizeError = NSLocalizedString(
+                "blazeEditAdView.image.imageSizeError",
+                value: "Please select an image with minimum dimensions of 500 * 500.",
```

Comment:
Nit: we should specify the unit for the image size for clarity.

```suggestion
                value: "Please select an image with minimum dimensions of 500 × 500 px.",
```

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/12470/commits/e0806346a0b0b918b91ec22e4d559076dc49cb41